package.json

```diff
@@ -17,6 +17,14 @@
     "test": "jest -- --coverage"
   },
   "dependencies": {
+    "@emotion/react": "11.7.1",
+    "@emotion/styled": "11.6.0",
+    "@fontsource/roboto": "4.5.1",
```

Comment:
Is this or importing the font in `index.html` necessary if you already use either of them?